appinfo/routes.php

```diff
@@ -343,6 +343,15 @@
 				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
+		[
+			'name' => 'Room#resendEmails',
+			'url' => '/api/{apiVersion}/room/{token}/participants/emails',
```

Comment:
```suggestion
			'name' => 'Room#resendInvitations',
			'url' => '/api/{apiVersion}/room/{token}/participants/resend-invitations',
```

Let's be prepared for non-email invites in the future